src/routes/contact-requests.ts

```diff
@@ -0,0 +1,32 @@
+import express, { Request, Response, Router } from 'express';
+import { ContactRequestsCollection } from '../collections/contact-requests';
+
+const contactRepository: ContactRequestsCollection = new ContactRequestsCollection();
+contactRepository.initContactRequestData();
+
+const router: Router = express.Router();
+export default router;
+
+// If a GET request is made to the "contact" route, return all saved contact requests.
+router.get('/', async (req: Request, res: Response) => {
+    const contactRequests = await contactRepository.returnAllContactRequests();
+    res.status(200).json(contactRequests);
```

Comment:
We don't _need_ to send `code`, `status`, and `statusText`. By all means - add them back in if you expect to use them in your UI code!

But we can access the code using [response.status](https://developer.mozilla.org/en-US/docs/Web/API/Response/status) in the UI code!

Reply:
Hmmm... I'm not super passionate about having those properties since they are being sent anyway. I think I had them in there from some boilerplate code I was looking at and, after starring at it too long, I just started to assume they were needed.